src/managers/CacheManager.js

```diff
@@ -0,0 +1,52 @@
+class CacheManager extends Map {
+    /**
+     * Creates a new CacheManager instance.
+     * @param {Number} ttl The time to live for the cache in minutes.
+     */
+    constructor(client, exports) {
+        super();
+
+        this.client = client;
+        this.exports = exports;
```

Comment:
Does this do anything? I feel like there's some implicit stuff going on that I don't understand yet 😄

Reply:
Fixed